TACC/Core-CMS-Custom
# bug/Refactor

Removed unused libraries and packages, rearranged the order to make packages more readable, and added apcd group check to submission view

## Overview
Quick refactor of some code to make it more readable and simplified.

…


## Changes
Only functional change is that  there is now a check to see if a user has an apcd role before they can view a submission. Everything else is clean-up of code that's unused.
…

## Testing

1. Make sure that you can view the the "Submit a File" page in the Submissions menu while all your roles are turned on. 
2. Use Django CMS to remove all your roles, including staff and superuser, and then see if you still access the "List Submissions" page in the Submissions menu.
3. Ping Frank to restore your roles.
4.  Review pages that you wrote and make sure deleted code is unnecessary.

## UI
